src/routes/+page.svelte

```diff
@@ -178,36 +178,36 @@
 				extensions: ['.mxl', '.musicxml', '.xml'],
 				description: $t('musicxml_files')
 			},
-			// {
-			// 	mimeTypes: ['audio/midi'],
-			// 	extensions: ['.mid', '.midi', '.kar'],
-			// 	description: $t('midi_files')
-			// },
-			// {
-			// 	mimeTypes: ['application/x-musedata'],
-			// 	extensions: ['.md'],
-			// 	description: $t('musedata_files')
-			// },
-			// {
-			// 	mimeTypes: ['application/x-capella'],
-			// 	extensions: ['.cap', '.capx'],
-			// 	description: $t('capella_files')
-			// },
-			// {
-			// 	mimeTypes: ['application/x-biab'],
-			// 	extensions: ['.mgu', '.sgu'],
-			// 	description: $t('bb_files') + $t('experimental')
-			// },
-			// {
-			// 	mimeTypes: ['application/x-overture'],
-			// 	extensions: ['.ove', '.scw'],
-			// 	description: $t('overture_score_writer_files') + $t('experimental')
-			// },
-			// {
-			// 	mimeTypes: ['application/x-bww'],
-			// 	extensions: ['.bmw', '.bww'],
-			// 	description: $t('bagpipe_music_writer_files') + $t('experimental')
-			// },
+			{
+				mimeTypes: ['audio/midi'],
+				extensions: ['.mid', '.midi', '.kar'],
+				description: $t('midi_files')
+			},
+			{
+				mimeTypes: ['application/x-musedata'],
+				extensions: ['.md'],
+				description: $t('musedata_files')
+			},
+			{
+				mimeTypes: ['application/x-capella'],
+				extensions: ['.cap', '.capx'],
+				description: $t('capella_files')
+			},
+			{
+				mimeTypes: ['application/x-biab'],
+				extensions: ['.mgu', '.sgu'],
+				description: $t('bb_files') + $t('experimental')
+			},
+			{
+				mimeTypes: ['application/x-overture'],
+				extensions: ['.ove', '.scw'],
+				description: $t('overture_score_writer_files') + $t('experimental')
+			},
+			{
+				mimeTypes: ['application/x-bww'],
+				extensions: ['.bmw', '.bww'],
+				description: $t('bagpipe_music_writer_files') + $t('experimental')
+			},
```

Comment:
I don't think webmscore ever supported these file types

Reply:
Does webmscore explicitly not support them? MuseScore supports them so that's why I added them

Reply:
https://github.com/LibreScore/webmscore/blob/v1.1.0/web-public/schemas.ts#L249-L257

Also in MuseScore 3 based version, it explicitly handles formats in https://github.com/LibreScore/MuseScore/blob/fd19dd923873148bf41dd2af60ccbc66de56e575/web/main.cpp#L125-L140

Reply:
MuseScore 4:
![image](https://user-images.githubusercontent.com/26418974/213129362-1c64595f-657b-4993-bfff-a98dfb69b6e1.png)

Reply:
MuseScore backup files changed from `.mscz,` to `.mscz~`, I'll have to review their changes

Reply:
> MuseScore backup files changed from `.mscz,` to `.mscz~`, I'll have to review their changes

I found this
https://github.com/LibreScore/webmscore/blob/v1.1.0/src/project/internal/projectconfiguration.cpp#L659-L665

Reply:
I'm guessing they changed the name due to the `,` not being supported in browsers: https://github.com/GoogleChromeLabs/browser-fs-access/issues/63